rust/src/measurements/laplace/float/pseudocode/make_scalar_float_laplace_cks20.py

```diff
@@ -0,0 +1,22 @@
+# type: ignore
+def make_scalar_float_laplace_cks20(input_domain, input_metric, scale: QO, k):
+    if scale.is_sign_negative():
+        raise ValueError("scale must not be negative")
+    
+    k, relaxation = get_discretization_consts(k)
+
+    if scale.is_zero():
+        def function(x: T):
+            return x
+    else:
+        def function(x: T):
+            return sample_discrete_laplace_Z2k(x, scale, k)
+    
+    return Measurement(
+        input_domain,
+        function,
+        input_metric,
+        MaxDivergence(QO),
+        privacy_map=laplace_map(scale, relaxation=relaxation)
+    )
```

Comment:
```suggestion
    return Measurement(
        input_domain=input_domain,
        function=function,
        input_metric=input_metric,
        output_metric=MaxDivergence(QO),
        privacy_map=laplace_map(scale, relaxation=relaxation)
    )
```
Most of the names are indicative already, but could be a little more readable. This is also how the `make_report_noisy_max_gumbel.py` does it.